platform.json

```diff
@@ -29,7 +29,7 @@
     "toolchain-gccarmnoneeabi": {
       "type": "toolchain",
       "owner": "platformio",
-      "version": "~1.90201.0"
+      "version": ">=1.90201.0,<1.90302.0"
```

Comment:
Can we add the `>=` to line 50 (framework-arduinopico) version?  I do normally release a new core every couple weeks and it seems like this would allow easier use for end users.  Since only 1 version of the core release can be uploaded to P.IO, there would otherwise be a few days breakage where the P.IO package was updated but a new PR updating the core release wasn't yet merged

Reply:
The above line is only for compilation with the ArduinoCore-mbed which I introduced due to https://github.com/esphome/esphome/pull/3284#issuecomment-1174982922, the toolchain for Arduino-Pico (`toolchain-rp2040-...`) is unaffected.

The "only 1 version" is a misuderstanding, PIO staff deleted the version (that's what I read from the comment above) to just have 1 version when the PR is merged -- there should be no problems with more versions when the PR is merged, and having an exact reference to the core + toolchain for Arduino-Pico should be morme stable.